### PR TITLE
Use ibm-odlm as the package name

### PR DESCRIPTION
**Pull request by horis233**
**What this PR does / why we need it**:

Update ODLM package name from `operand-deployment-lifecycle-manager` to `ibm-odlm`.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: bundle/manifests/operator.ibm.com_operandrequests.yaml [hunk 1]
```diff
@@ -72,9 +73,19 @@ spec:
...
+                          description: InstanceName is used when users want to deploy multiple custom resources It is the name of the custom resource
+                          type: string
+                        kind:
+                          description: Kind is used when users want to deploy multiple custom resources Kind identifies the kind of the custom resource
+                          type: string
                         name:
                           description: Name of the operand to be deployed
                           type: string
+                        spec:
+                          description: Spec is used when users want to deploy multiple custom resources It is the configuration map of custom resource
```
Comment (Daniel-Fan) [inline bundle/manifests/operator.ibm.com_operandrequests.yaml:86]: Would you mind to add commas here? :-)

File: deploy/olm-catalog/operand-deployment-lifecycle-manager/1.4.0/operator.ibm.com_operandrequests_crd.yaml [hunk 2]
```diff
@@ -72,9 +73,19 @@ spec:
...
+                          description: InstanceName is used when users want to deploy multiple custom resources It is the name of the custom resource
+                          type: string
+                        kind:
+                          description: Kind is used when users want to deploy multiple custom resources Kind identifies the kind of the custom resource
+                          type: string
                         name:
                           description: Name of the operand to be deployed
                           type: string
+                        spec:
+                          description: Spec is used when users want to deploy multiple custom resources It is the configuration map of custom resource
```
Comment (Daniel-Fan) [inline deploy/olm-catalog/operand-deployment-lifecycle-manager/1.4.0/operator.ibm.com_operandrequests_crd.yaml:86]: Would you mind to add commas here as well?  :-)

**Review by Daniel-Fan — CHANGES_REQUESTED** [review]

**Comment by horis233** [comment]
@Daniel-Fan Nice catch. I add the punctuation to OperandRequest comments. Could you check the other comments in the ODLM repo and add punctuations for them if they don't have punctuations.

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**